src/util/state-management.js

```diff
@@ -77,19 +78,29 @@ export function getStatefulPropsContext(reducers, { getState, setState }) {
  * Generates a root selector from a tree of selectors
  * @param {Object} selectors - a tree of selectors
  * @returns {function} root selector that when called with state, calls each of
- * the selectors in the tree with the state local to that selector
+ * the selectors in the tree with the state local to that selector.
+ *
+ * This function is memoized because it's recursive, and we want it to reuse
+ * the functions created in the recursive reduce (because those functions are
```

Comment:
I'm not sure I'm following the part in parenthesis. Which other functions are also memoized? The selector functions? Is that something we do, or a part of how reselect works?